oap-server/server-receiver-plugin/skywalking-ebpf-receiver-plugin/src/main/java/org/apache/skywalking/oap/server/receiver/ebpf/provider/handler/EBPFProfilingServiceHandler.java

```diff
@@ -85,7 +85,7 @@ public void queryTasks(EBPFProfilingTaskQuery request, StreamObserver<Commands>
         try {
             // find exists process from agent
             final List<Process> processes = metadataQueryDAO.listProcesses(null, null, agentId,
-                    ProfilingSupportStatus.SUPPORT_EBPF_PROFILING, 0, 0);
+                    null, 0, 0);
```

Comment:
Why we need all processes(including not profiling-able processes?)

Reply:
Because the network profiling doesn't require the process could be profiling to generate the flame graph. So It depends on the target type of the profiling task.

Reply:
Then, I feel whether we should adjust the APIs? (`IMetadataQueryDAO#listProcesses`). We are randomly using `null` and empty values, which mean differently. Let's make the meaning of storage APIs clear?